corpus/pages/views.py

```diff
@@ -42,5 +46,40 @@ def sig(request, sig_name):
 
     return render(request, "pages/sig.html", args)
 
+
+def team(request):
+    compsoc_members = ExecutiveMember.objects.filter(
+        (Q(core__isnull=True) | Q(core=None))
+        & (Q(user__faculty__isnull=True) | Q(user__faculty=None))
+        & Q(sig__name="CompSoc")
+    )
+    diode_members = ExecutiveMember.objects.filter(
```

Comment:
Two things you can do to solve this:
1. Duplicate core members in the members list. This is fine, as although they are in core, they are still members, so should be okay.
2. Get all the core posts, and then remove those from all the members in that SIG.

Also, you'll have to make sure that you query for this particular year. You can probably make a configuration where you define what the current academic year is, and then use that for queries to get "current" people.

Reply:
Can I like check if their term_end is null or not? If term_end is null, then display, otherwise don't?